tests/e2e/utils/helpers.go

```diff
@@ -81,7 +81,8 @@ func newHTTPClient() http.Client {
 			Transport: &http.Transport{
```

Comment:
We probably want to specify a timeout on the `http.Client` too.

```go
Timeout: time.Second * 15,
```

Reply:
Can this be in another PR? Rerunning this takes 1h and we need to cut a new RC.

Reply:
💯